ncf/SPECS/ncf.spec

```diff
@@ -49,8 +49,6 @@ Source1: rudder-sources
 BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 BuildArch: noarch
 
-#BuildRequires: gcc
```

Comment:
Seriously, let's clean that up. This package only drops a folder on the target machine :)